gui/guihtml/src/TGHtmlUri.cxx

```diff
@@ -350,7 +350,7 @@ char *TGHtml::ResolveUri(const char *zUri)
             zBuf[i] = 0;
          }
          // coverity[secure_coding]
-         strcat(zBuf, term->fZPath);
+         strlcat(zBuf, term->fZPath, sizeof(zBuf));
```

Comment:
Here is wrong, you need to remember allocated size and use it here

Reply:
Same comment